cmd/rollkit/commands/run_node_test.go

```diff
@@ -84,6 +85,7 @@ func TestParseFlags(t *testing.T) {
 		{"DAMempoolTTL", nodeConfig.DAMempoolTTL, uint64(10)},
 		{"DANamespace", nodeConfig.DANamespace, "namespace"},
 		{"DAStartHeight", nodeConfig.DAStartHeight, uint64(100)},
+		{"DAKeyName", nodeConfig.DAKeyName, "my_celes_key"},
```

Comment:
Where credible neutrality? :sob:

IMHO empty default (aka. user has to provide someting, or da implementation can provide da-specific default) is a better solution.

Reply:
Agreed! I commented this but I think since I changed it to KeyName it has dissapeared 😶‍🌫️ 

Should I use `null` here?